### ISSUE TITLE
Transform Manager build fail

### ISSUE THREAD

**Issue by NathanaelThompson** [v3n/altertum#17]
Before attempting this build, I ran the following commands in the git shell: 

> git fetch origin
> git reset --hard origin/master
> git submodule update --recursive
> git pull

Then I ran the genie tool, and ran a clean build of the project.  This is where I am now.  I was going to attach the file to this issue, but GitHub keeps claiming something went really wrong.  So, I am instead dropping the error log into slack.  I am assuming this, and the fact that I can't add tags extends from the fact that I am not a contributor for this GitHub project.

**Comment by v3n** [comment]
This could be a most vexing parse issue, looking into it now.

**Comment by v3n** [comment]
This is a Visual Studio namespace lookup bug, for example, this is valid C++ for clang and gcc:

```
namespace A {
    struct X  {};

    namespace B {
        struct Y {
            void fun (X);
        };
    }
}

using namespace A::B;

void Y::fun(X) {}

int main () {
    A::B::Y y;
}
```

However, this does not compile on Visual Studio, requiring explicit namespace specifiers like so:

```
namespace A {
    struct X  {};

    namespace B {
        struct Y {
            void fun (X);
        };
    }
}

void A::B::Y::fun(X) {}

int main () {
    A::B::Y y;
}
```

Will evaluate and fix tonight.

**Comment by v3n** [comment]
Resolved in 3296eba1a36c1d91ce09cfe000dc53633dcf8690
